api/src/core/sql/schema/v1.schema.sql

```diff
@@ -529,20 +530,29 @@ create table word_range_tags_votes(
 
 -- SITE TEXT -------------------------------------------------------------
 
-create table original_site_text (
-  original_site_text_id bigserial primary key,
+create table site_text_words (
+  site_text_id bigserial primary key,
   word_id bigint not null references words(word_id),
   created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   created_by bigint not null references users(user_id)
 );
 
-create table orignal_site_text_votes(
-  original_site_text_vote_id bigserial primary key,
+create table site_text_phrases (
+  site_text_id bigserial primary key,
+  phrase_id bigint not null references phrases(phrase_id),
+  created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  created_by bigint not null references users(user_id)
+);
+
+create table site_text_translation_votes(
+  site_text_vote_id bigserial primary key,
   user_id bigint not null references users(user_id),
-  original_site_text_id bigint not null references original_site_text(original_site_text_id),
+  from_id bigint not null,
+  to_id bigint not null,
+  from_type_is_word bool not null, -- true = word, false = phrase
```

Comment:
we need `from_type_is_word` and `to_type_is_word`
a site text string may be a word or phrase

Reply:
I changed